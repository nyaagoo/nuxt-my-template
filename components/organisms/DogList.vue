<template lang="pug">
  .dog-list
    ol.dog-list
      li.dog-list-item(v-for="dog in dogList" :key="dog.name") 
        DogListItem(:dog="dog" :greet="shjua")
</template>
<script lang="ts">
import { Component, Vue } from 'vue-property-decorator'
import DogListItem from '~/components/molecules/DogListItem.vue'
import { dogStore } from '~/store/index'

@Component({
  name: 'dog-list',
  components: { DogListItem }
})
export default class DogList extends Vue {
  get dogList(): { name: string; img: string }[] {
    return dogStore.dogList
  }
}
</script>
<style lang="stylus" scoped>
.dog-list
  background #fff
</style>
