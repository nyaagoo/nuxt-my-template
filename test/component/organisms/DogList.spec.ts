import { mount } from '@vue/test-utils';
import DogList from '~/components/organisms/DogList.vue';

describe('DogList', () => {
  beforeAll(() => {});
  test('DogList.vue is a Vue instance', () => {
    const wrapper = mount(DogList);
    expect(wrapper.isVueInstance).toBeTruthy();
  });
});
