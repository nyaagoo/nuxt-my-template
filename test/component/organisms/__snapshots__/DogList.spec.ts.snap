// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DogList Snapshot 1`] = `
<div
  class="dog-list"
>
  <ul />
</div>
`;
